integration/common.go

```diff
@@ -47,8 +47,8 @@ var (
 
 func initImages(imageListFile string) {
 	imageList = ImageList{
-		Alpine:           "docker.io/library/alpine:latest",
-		BusyBox:          "docker.io/library/busybox:latest",
+		Alpine:           "ghcr.io/containerd/alpine:3.14.0",
```

Comment:
This image is only used in a single test; https://github.com/containerd/containerd/blob/cf600abecc27200a3c0e1415cd1f6c325eb05edf/integration/restart_test.go#L142-L145


And from that test, I _think_ the only reason it's pulled is "to have some images before restarting, and check that the images are still there after restarting);

https://github.com/containerd/containerd/blob/cf600abecc27200a3c0e1415cd1f6c325eb05edf/integration/restart_test.go#L182-L184


If that's the only reason, we might as well pull any of the other images from this list (e.g. `k8s.gcr.io/pause:3.5`, `gcr.io/k8s-cri-containerd/volume-copy-up:2.0`).

Reply:
I also noticed we're already using some mirrored images; https://github.com/containerd/containerd/blob/ba70277fb75e2976c4e9e817bf02e20bcb6a1fa1/integration/client/client_unix_test.go#L35

https://github.com/containerd/containerd/blob/ba70277fb75e2976c4e9e817bf02e20bcb6a1fa1/integration/client/container_linux_test.go#L58


Would it make sense to use those here as well? Then we could change those above to;

```go
testImage    = integration.GetImage(integration.BusyBox)
```

and

```go
var testUserNSImage = integration.GetImage(integration.Alpine)
```